`vite`: Handle error when importing sku render entrypoint (#1551)
seek-oss/sku: .changeset/shiny-peas-clean.md

`vite`: Improve error messaging during `sku build`

> The path of the first route that fails to render will now be logged, along with a stack trace to aid debugging. Errors that occur before rendering will also now be logged with a stack trace.

Reviewer: Could combine the sentences 
```suggestion
Pre-render errors and failures in the first errored route will now be logged with stack traces for easier debugging.
```
